scrapyd_k8s/tests/integration/test_api.py

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python3
+from datetime import datetime
 import os
-import pytest
 import requests
 import time
 
+from scrapyd_k8s.api import launcher
+from scrapyd_k8s.launcher.k8s import K8s
+from scrapyd_k8s.utils import TIME_FORMAT
 
```

Comment:
Ok, you really cannot import anything from the project's code here. Please keep this an integration test that tests the API. Even `TIME_FORMAT` is something to hardcode here - if it changes in the code, this is an API breakage, so we want to catch this in the tests.

Reply:
I think we will need to document this in a comment on top of the file, that this test is meant to be run against a running instance of scrapyd-k8s, and is a _standalone_ test (and does and must not reference scrapyd-k8s code). We are testing scrapyd-k8s REST API conformity and behaviour, _not_ the Python code.